InvenTree/company/models.py

```diff
@@ -266,10 +266,19 @@ class Address(models.Model):
         link: External link to additional address information
     """
 
+    def __str__(self):
+        """Defines string representation of address to supple a one-line to API calls"""
+        return ", ".join([self.line1,
+                          self.line2,
+                          self.postal_code,
+                          self.postal_city,
+                          self.province,
+                          self.country])
```

Comment:
Question: How does this handle empty line2 etc?

Reply:
That's an edgecase I haven't explored yet. But as far as I see, forms will commit an empty field as "". Concat on an empty string will just result in an empty string entry. In this case, it might indeed be wise to concat line1 and line2 if `line2 != ""` and use the resulting string (or just line1) as input to the join.